src/content/blog/v1.0_release.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "AutoEmulate v1.0.0 release"
+pubDate: "2025-07-31"
+description: "Updates on new features in v1.0.0 release"
+draft: true
+author: "AutoEmulate contributors"
+release: "v1.0.0"
+---
+
+🎉 We're excited to announce AutoEmulate's first major release with v1.0.0 🎉
+
+<br/><br/>
+This release sees the package fully integrated into the PyTorch ecosystem; PyTorch is one of the most popular open-source deep learning frameworks and one of the bedrocks of modern machine learning with Python.
+
+This release also contains new features, including methods for uncertainty quantification;  ensemble-based emulator models, which output predictions with uncertainty, and Bayesian model calibration, which enables inference of parameters likely to have generated observed data with uncertainty.
+
+<br/><br/>
+If you'd like to discuss any of our work on AutoEmulate or AI for Physical Systems, feel free to reach out to us at ai4physics@turing.ac.uk
+
+## What's new
+
+### PyTorch refactor and package redesign
+
+AutoEmulate was originally built on top of the scikit-learn framework, which is a great library for traditional machine learning tasks. However, as we moved towards more complex models and larger datasets, we found that we needed the flexibility and performance that PyTorch provides. The new PyTorch backend allows us to leverage GPU acceleration, making training and inference much faster. The PyTorch focus also allows AutoEmulate to integrate with other tools in the broader ecosystem, enabling end-to-end emulation workflows.
+
+### A new set of Emulator models
+
+AutoEmulate v1.0.0 includes a new set of Emulator models with a variety of different architectures. Some of these are wrapped implementations from PyTorch, scikit-learn and elsewhere, but we also have a number of custom implementations including ensemble models where we quantify uncertainty by looking at the distribution over outputs. The table below summarises which emulators available in this release are implemented in PyTorch, support multi-output emulation (MO), use auto-differentiation (AD) for training, or support uncertainty quantification (UQ).
```

Comment:
I wonder if we can rephrase this section (and maybe the table as well) slightly. I think we should say that all our (main) models are built on PyTorch  and then say that we've also retained support for some of the models in the original version (because we've been told ppl find them useful) but that using these naturally looses the PyTorch benefits

Reply:
Thanks @radka-j this has been rephrased, I recently also rearranged the table so the non-PyTorch ones are at the bottom and the Emulators with the most features are nearest the top